src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
So pfSense needs a complete overhaul to put content of all critical files into the config.xml and rewrite them at boot up to ensure they have the correct (assumed to be correct form config.xml) content.  Including the CA bundle.  That is what the argument you are making amounts to.

The contents of these cert files is no more and no less critical that that of the CA bundle.  If this is necessary and justified for these files to be rewritten from config at every boot up then the same is true for the CA bundle.  It can be corrupted just as easily as these individual cert files.

The reason I can't admit that I'm wrong is because I'm not.  Regardless of how you manage to convince it to be done.  My systems will not be rewriting all the CA's at boot.